modules/synacormediaBidAdapter.js

```diff
@@ -68,10 +68,6 @@ export const spec = {
         seatId = bid.params.seatId;
       }
       const tagIdOrplacementId = bid.params.tagId || bid.params.placementId;
```

Comment:
The variable name is still `tagIdOrplacementId` (note the lower case `p`), while later it has been corrected to `tagIdOrPlacementId` (upper-case `P`).

Reply:
@musikele I fixed the typo